meta-intel-extras/recipes-kernel/linux/linux-intel_%.bbappend

```diff
@@ -1,10 +1,12 @@
 #
 #   Copyright (C) 2016 Pelagicore AB
-#   Copyright (C) 2018 Luxoft Sweden AB
+#   Copyright (C) 2018-2019 Luxoft Sweden AB
```

Comment:
You really don't need to add this, even if you find something with 2017. If Luxoft Sweden AB is missing, then I can agree it needs to be added.